mpa/modules/models/heads/custom_hierarchical_non_linear_cls_head.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import normal_init, constant_init, build_activation_layer
+
+from mmcls.models.builder import HEADS, build_loss
+from mmcls.models.heads import MultiLabelClsHead
+
+
+@HEADS.register_module()
+class CustomHierarchicalNonLinearClsHead(MultiLabelClsHead):
+    """Custom Linear classification head for multilabel task.
```

Comment:
I think description for this function is not match
Also, NonLinearClsHead is not intuitive. How about MLPHead?

Reply:
I don't agree with MLPHead, The difference between LinearClsHead and NonLinearClsHead cannot be summarized by MLPHead. But I'll think about more intuitive name. Thank you for your suggestion.